src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSink.java

```diff
@@ -153,18 +153,11 @@ private static void writeReadsSingle(
         // Set the header on the main thread.
         SparkBAMOutputFormat.setHeader(header);
 
-        // The expected format for writing is JavaPairRDD where the key is ignored and the value is a
-        // SAMRecordWritable. We use GATKRead as the key, so we can sort the reads before writing them to a file.
-        JavaPairRDD<GATKRead, SAMRecordWritable> rddSamRecordWriteable = rddReads.mapToPair(gatkRead -> {
-            SAMRecord samRecord = gatkRead.convertToSAMRecord(header);
-            SAMRecordWritable samRecordWritable = new SAMRecordWritable();
-            samRecordWritable.set(samRecord);
-            return new Tuple2<>(gatkRead, samRecordWritable);
-        });
+        JavaPairRDD<GATKRead, Void> rddReadPairs = rddReads.mapToPair(gatkRead -> new Tuple2<>(gatkRead, (Void) null));
 
         // do a total sort so that all the reads in partition i are less than those in partition i+1
-        final JavaPairRDD<GATKRead, SAMRecordWritable> out =
-                totalSortByKey(rddSamRecordWriteable, new ReadCoordinateComparator(header), GATKRead.class);
+        final JavaPairRDD<GATKRead, Void> out =
```

Comment:
Could you add a comment here explaining why it's void?  I feel like someone is going to be confused in the future.